system-test/bigquery.ts

```diff
@@ -1701,7 +1701,8 @@ describe('BigQuery', () => {
     const deleteDatasetPromises = datasets
       .filter(async dataset => {
         const [metadata] = await dataset.getMetadata();
-        return isResourceStale(parseInt(metadata.creationTime));
+        const creationTime = parseInt(metadata.creationTime);
```

Comment:
You almost always want to use `Number(metadata.creationTime)` over `parseInt`.  ParseInt has some weird behaviors around radix detection and string parsing.  This thread has a great explanation:
https://stackoverflow.com/questions/4090518/what-is-the-difference-between-parseint-and-number

Reply:
This is really helpful, thank you!